Cargo.toml

```diff
@@ -28,3 +28,10 @@ opt-level = 3
 # Those are here to dedupe winapi since mio is still using winapi 0.2.
 mio = { git = "https://github.com/servo/mio.git", branch = "servo" }
 iovec = { git = "https://github.com/servo/iovec.git", branch = "servo" }
+
+[patch."https://github.com/servo/media"]
+servo-media = { git = "https://github.com/ferjm/media", branch = "suspend" }
+servo-media-gstreamer = { git = "https://github.com/ferjm/media", branch = "suspend" }
+servo-media-audio = { git = "https://github.com/ferjm/media", branch = "suspend" }
+servo-media-player = { git = "https://github.com/ferjm/media", branch = "suspend" }
+servo-media-streams = { git = "https://github.com/ferjm/media", branch = "suspend" }
```

Comment:
this is temporal, right?

suspend branch is behind current media master

Reply:
Yes. I will remove the patch as soon as https://github.com/servo/media/pull/310 is merged